specifyweb/businessrules/agent_rules.py

```diff
@@ -6,20 +6,45 @@
 def agent_delete_blocked_by_related_specifyuser(agent):
     try:
         Specifyuser.objects.get(agents=agent)
```

Comment:
Remove this line. It's redundant with the next line

Reply:
Fixed in 44f6b8e